src/routes/committees/website/+page.svelte

```diff
@@ -41,6 +45,8 @@
 	</Li>
 	<Li>Have common a goal and outline for a feature to be completed.</Li>
 </List>
+</div>
+</div>
 <Heading tag="h3" class="text-center" customSize="text-lg font-extrabold m-4  md:text-xl lg:text-xl"
```

Comment:
For the resources on this page, I would like to have it look like it does on the picture in the task, not underneath. Also don't forget to prettify it. If you don't know how to prettify it make sure you have the extension and press alt shift f